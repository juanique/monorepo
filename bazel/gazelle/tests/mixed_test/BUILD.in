load("//bazel/ts:defs.bzl", "ts_library")

# This file intentionally empty - Gazelle should generate the appropriate rule
