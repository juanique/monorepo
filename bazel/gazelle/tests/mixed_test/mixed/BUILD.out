load("//bazel/js:defs.bzl", "ts_binary")

ts_binary(
    name = "mixed",
    srcs = [
        "helper.js",
        "main.ts",
    ],
)
